src/components/AnonymousChat/BaseChatItem.tsx

```diff
@@ -7,7 +7,7 @@ import BaseSystemChat from './BaseChatSystem';
 import ChatItemMyTextV2 from './child/ChatItemMyTextV2';
 import ChatItemTargetText from './child/ChatItemTargetText';
 import dimen from '../../utils/dimen';
-import useChatScreenHook from '../../hooks/screen/useChatScreenHook';
+import useChatUtilsHook from '../../hooks/core/chat/useChatUtilsHook';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been changed from `useChatScreenHook` to `useChatUtilsHook`. Ensure that the new hook provides all the necessary functionalities required in this component.
```diff
- import useChatScreenHook from '../../hooks/screen/useChatScreenHook';
+ import useChatUtilsHook from '../../hooks/core/chat/useChatUtilsHook';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->